sdk/scheduler/src/main/java/com/mesosphere/sdk/specification/DefaultServiceSpec.java

```diff
@@ -175,11 +177,24 @@ private Comparator() {
         }
 
         @Override
-        public boolean equals(ServiceSpec first, ServiceSpec second) {
-            return EqualsBuilder.reflectionEquals(first, second);
+        public boolean equals(ServiceSpec current, ServiceSpec proposed) {
```

Comment:
See: DefaultConfigurationUpdater.java:L117 
You can not change placement rule at all if this PR is in. Am I missing something here?